validation/src/main/scala/hmda/validation/rules/lar/syntactical/S270.scala

```diff
@@ -0,0 +1,23 @@
+package hmda.validation.rules.lar.syntactical
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.context.ValidationContext
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax.PredicateOps
+import hmda.validation.dsl.Result
+import hmda.validation.rules.{ EditCheck, IfYearPresentIn }
+
+object S270 {
+  def inContext(ctx: ValidationContext): EditCheck[LoanApplicationRegister] = {
+    IfYearPresentIn(ctx) { new S270(_) }
+  }
+}
+
+class S270 private (year: Int) extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "S270"
```

Comment:
The method does override a method on the abstract class, though. Why do you think I should remove it?